apps/trading/pages/_app.page.tsx

```diff
@@ -30,79 +27,73 @@ function AppBody({ Component, pageProps }: AppProps) {
 
   return (
     <div className="h-full dark:bg-black dark:text-white-60 bg-white relative z-0 text-black-60 grid grid-rows-[min-content,1fr]">
-      <div className="flex items-stretch border-b-[7px] border-vega-yellow">
-        <Navbar />
-        <div className="flex items-center gap-4 ml-auto mr-8">
-          <VegaWalletConnectButton
-            setConnectDialog={(open) => {
-              store.setVegaWalletConnectDialog(open);
-            }}
-            setManageDialog={(open) => {
-              store.setVegaWalletManageDialog(open);
-            }}
-          />
-          <ThemeSwitcher onToggle={toggleTheme} className="-my-4" />
+      <AppLoader>
+        <div className="flex items-stretch border-b-[7px] border-vega-yellow">
+          <Navbar />
+          <div className="flex items-center gap-4 ml-auto mr-8">
+            <VegaWalletConnectButton
+              setConnectDialog={(open) => {
+                store.setVegaWalletConnectDialog(open);
+              }}
+              setManageDialog={(open) => {
+                store.setVegaWalletManageDialog(open);
+              }}
+            />
+            <ThemeSwitcher onToggle={toggleTheme} className="-my-4" />
+          </div>
         </div>
-      </div>
-      <main data-testid={pageProps.page}>
-        {/* @ts-ignore conflict between @types/react and nextjs internal types */}
-        <Component {...pageProps} />
-      </main>
-      <VegaConnectDialog
-        connectors={Connectors}
-        dialogOpen={store.vegaWalletConnectDialog}
-        setDialogOpen={(open) => store.setVegaWalletConnectDialog(open)}
-      />
-      <VegaManageDialog
-        dialogOpen={store.vegaWalletManageDialog}
-        setDialogOpen={(open) => store.setVegaWalletManageDialog(open)}
-      />
-      <NetworkSwitcherDialog
-        dialogOpen={store.vegaNetworkSwitcherDialog}
-        setDialogOpen={(open) => store.setVegaNetworkSwitcherDialog(open)}
-        onConnect={({ network }) => {
-          if (VEGA_NETWORKS[network]) {
-            window.location.href = VEGA_NETWORKS[network];
-          }
-        }}
-      />
+        <main data-testid={pageProps.page}>
+          {/* @ts-ignore conflict between @types/react and nextjs internal types */}
+          <Component {...pageProps} />
+        </main>
+        <VegaConnectDialog
+          connectors={Connectors}
+          dialogOpen={store.vegaWalletConnectDialog}
+          setDialogOpen={(open) => store.setVegaWalletConnectDialog(open)}
+        />
+        <VegaManageDialog
+          dialogOpen={store.vegaWalletManageDialog}
+          setDialogOpen={(open) => store.setVegaWalletManageDialog(open)}
+        />
+        <NetworkSwitcherDialog
+          dialogOpen={store.vegaNetworkSwitcherDialog}
+          setDialogOpen={(open) => store.setVegaNetworkSwitcherDialog(open)}
+          onConnect={({ network }) => {
+            if (VEGA_NETWORKS[network]) {
+              window.location.href = VEGA_NETWORKS[network] ?? '';
```

Comment:
chore (blocking): Should probably use nextjs's router.push function here.

Reply:
totally true, changed this, thanks!